src/applications/pensions/config/form.js

```diff
@@ -1432,6 +1433,12 @@ const formConfig = {
           CustomPage: HomeAcreageValueInput,
           CustomPageReview: null,
         },
+        landMarketable: {
+          title: 'land marketable',
```

Comment:
This title will need to be sentence cased since it can show up on the Review page
```suggestion
          title: 'Land marketable',
```